test/features/deploy_kogito_runtime.feature

```diff
@@ -306,14 +308,17 @@ Feature: Deploy Kogito Runtime
   Scenario: Deploy <example-service> with Maven profile <profile> with events using external Kafka
     Given Kogito Operator is deployed with Infinispan and Kafka operators
     And Kafka instance "external-kafka" is deployed
-    And Install Kogito Data Index with 1 replicas with configuration:
-      | kafka | externalURI | external-kafka-kafka-bootstrap:9092 |
+    And Install Infinispan Kogito Infra "infinispan" within 5 minutes
+    And Infinispan instance "kogito-infinispan" has 1 pod running within 5 minutes
+    And Install Kafka Kogito Infra "external-kafka" connected to resource "external-kafka" within 5 minutes
+    When Install Kogito Data Index with 1 replicas with configuration:
```

Comment:
I know it doesn't matter functionally but here it should probably start with `And`.